fv3core/fv3core/stencils/fv_dynamics.py

```diff
@@ -436,85 +373,137 @@ def _compute(
                 # issue is that set_val in map_single expects a 3D field for the
                 # "surface" array
                 if __debug__:
-                    if self.comm.rank == 0:
+                    if self.comm_rank == 0:
                         print("Remapping")
-                with timer.clock("Remapping"):
-                    self._lagrangian_to_eulerian_obj(
-                        tracer_storages,
-                        state.pt,
-                        state.delp,
-                        state.delz,
-                        state.peln,
-                        state.u,
-                        state.v,
-                        state.w,
-                        state.ua,
-                        state.va,
-                        state.cappa,
-                        state.q_con,
-                        state.qcld,
-                        state.pkz,
-                        state.pk,
-                        state.pe,
-                        state.phis,
-                        state.te0_2d,
-                        state.ps,
-                        state.wsd,
-                        state.omga,
-                        self._ak,
-                        self._bk,
-                        self._pfull,
-                        state.dp1,
-                        self._ptop,
-                        constants.KAPPA,
-                        constants.ZVIR,
-                        last_step,
-                        state.consv_te,
-                        state.bdt / state.k_split,
-                        state.bdt,
-                        state.do_adiabatic_init,
-                        NQ,
-                    )
+                self._lagrangian_to_eulerian_obj(
```

Comment:
Are we okay with not having timer inside fvdynamics? I think at the time of orchestrating, I skipped over the timer wrapper for simplicity. I think we want it so that we can still get the numbers for regular backends.

Reply:
We are not ok, it is on my list, It requires me to fix the `with` statement in DaCe first. I was doing to do that in `delnflux`

Reply:
Okay, I think it'll be good to be part of this PR so that we can continue to collect timing. But if you think that will take some extra work and want to roll it in after, I think it's also fine.

Reply:
I am unhappy with it, but for now it requires me to change DaCe or do a start/end callback, both of which are painful. So my plan is to put it has a MUST DO before merge on the delnflux PR so we can move forward with the rest of integration in the meantime